src/components/client-list/client-list.test.tsx

```diff
@@ -14,6 +14,25 @@ const testClients = [
   },
 ];
 
+jest.mock('gatsby-plugin-react-i18next', () => ({
```

Comment:
It's best practice to put all functions inside the corresponding jest lifecycle method. In this case, the mocking should be put into `beforeAll()` or `beforeEach`.

https://jestjs.io/docs/setup-teardown